extension/src/engine/ws-client.ts

```diff
@@ -42,10 +42,30 @@ export const WebSocketClientProvider = (webSocketUrl: URL) => {
 };
 
 const logMessage = (entry: RuntimeLogEntry) => {
-  const logMessage = `[${entry.timestamp}][${entry.level}][${entry.project}] ${entry.message}`;
```

Comment:
do you receive logs from the past? If so, the timestamp created by vscode would not match the actual one. But I'm fine with this mismatch.